code-wars/tutor-code-wars/mccgrey-codewars/8ky-century-finder.js

```diff
@@ -0,0 +1,47 @@
+/*
+The first century spans from the year 1 up to and including the year 100, the second century - from the year 101 up to and including the year 200, etc.
+
+Task
+Given a year, return the century it is in.
+
+Examples
+1705 --> 18
+1900 --> 19
+1601 --> 17
+2000 --> 20
+
+*/
+
+
+
+function century(year) {
+  
+    return Math.ceil(year/100); //Use inbuilt Math functions in javascript for one-line answer
+  }
+  
+  console.log(century(1601));
+
+
+//Math.ceil function always rounds a number up to the next largest whole number or integer.
+
+
+
+
+
+/*
+function century(year) {
+
+    let century = 0; //declare a variable
+    
+    for(let i = 0; i < year; i++) {
+      if(i % 100 == 0) {
+        century++;
+      }
+    }
+    return century;
+  }
+
+  console.log(century(2000));
+
```

Comment:
- [ ] Kindly remove commented code from solution, it is often considered a good practice 👍